docs/dev/pos/longpolling.rst

```diff
@@ -2,88 +2,92 @@
  POS Longpolling
 =================
 
-It is a custom odoo module made by `IT-Projects LLC, <https://it-projects.info>`__ which provides instant updates to the POS interfaces from backend.
+It is a custom odoo `module <https://github.com/it-projects-llc/pos-addons/tree/12.0/pos_longpolling>`__ made by `IT-Projects LLC, <https://it-projects.info>`__ which allows sending instant updates to the POS interfaces from backend.
 
-Let take the **example** of the ``Sync Partners in POS`` module.
+It provides following methods in *Backend side*:
 
-* On each record update (needed to be sent) in ``res.partner`` model in ``send_field_updates``  `method. <https://github.com/it-projects-llc/pos-addons/blob/907b16cc3a4ea613bf4fc81891a03739405e57a7/pos_partner_sync/models/res_partner.py#L39-L43::>`__
-* It uses ``send_to_all_poses`` `method <https://github.com/it-projects-llc/pos-addons/blob/907b16cc3a4ea613bf4fc81891a03739405e57a7/pos_partner_sync/models/res_partner.py#L43>`__.
+* **self.env['pos.config].send_to_all_poses(channel_name, data)**:broadcasts messages to all opened POSes  (see `example <https://github.com/it-projects-llc/pos-addons/blob/28d2b00bfd3f5d09bb65d5bf3245a6b87ed1d67b/pos_longpolling/models/pos_longpolling_models.py#L49-L53>`__)
```

Comment:
It's better to use ``code`` formatting here.

Also, don't miss space after colon ``:``